automationtestapp/src/main/java/com/microsoft/aad/automation/testapp/MainActivity.java

```diff
@@ -192,6 +194,14 @@ private void processReadCacheRequest() {
         return allItems;
     }
 
+    private boolean isCommonCacheUsed(){
```

Comment:
@shoatman Please review the code and let me know if there is any other scenerio where we use common cache apart from this? From `TokenCacheAccessor` I see we update `mUseCommonCache` to true only in this case.

Reply:
We probably need something based on version rather than on whether the default token cache is being used.  If we want to test versions of ADAL that do not contain common cache... for example.

Reply:
As discussed, I looking for a specific class in common using reflection in addition to the default token cache to determine if we are saving using common